model_analyzer/config/generate/optuna_run_config_generator.py

```diff
@@ -159,27 +224,38 @@ def _create_objective_based_run_config(self) -> RunConfig:
         model_run_config = self._create_model_run_config(
             model=self._models[0],
             model_config_variant=model_config_variant,
+            trial_objectives=trial_objectives,
         )
 
         run_config.add_model_run_config(model_run_config=model_run_config)
 
         return run_config
 
-    def _create_parameter_combo(self) -> Dict[str, Any]:
-        # TODO: TMA-1925: Use SearchParameters here
-        param_combo: Dict["str", Any] = {}
-        param_combo["dynamic_batching"] = {}
+    def _create_parameter_combo(
+        self, trial_objectives: TrialObjectives
+    ) -> ParameterCombo:
+        param_combo: ParameterCombo = {}
+        param_combo["dynamic_batching"] = []
```

Comment:
be careful with hardcoding these. If we are moving forward we need full flexibility. Some models don't support dynamic batching.

Reply:
I've added a todo + story for this.